src/server/pkg/storage/chunk/metadata.go

```diff
@@ -103,17 +105,21 @@ func (s *postgresStore) Delete(ctx context.Context, chunkID ID) error {
 	return err
 }
 
-// SetupPostgresStore sets up tables in db
-func SetupPostgresStore(db *sqlx.DB) {
-	db.MustExec(schema)
-}
-
-const schema = `
-	CREATE SCHEMA IF NOT EXISTS storage;
-
-	CREATE TABLE storage.chunks (
+// SetupPostgresStoreV0 sets up tables in db
+func SetupPostgresStoreV0(ctx context.Context, tableName string, tx *sqlx.Tx) error {
```

Comment:
This is how I was thinking of parameterize the table locations.  But it doesn't seem like we need this right now and I think it harms the readability of the SQL so this is more just to get feedback on this method. We should probably pull it out before merging.

Reply:
Discussed offline, not parameterizing table names for now.